src/compiler/FileHandler/PosixFileHandler.ts

```diff
@@ -2,14 +2,17 @@ import { IFileHandler } from "../IFileHandler";
 import * as path from "path";
 import * as fs from "fs";
 
+// This class replaces upstream's DefaultFileHandler.
 export class PosixFileHandler implements IFileHandler {
```

Comment:
I made some changes to POSIX file handler, because I needed a file handler that was closer to `DefaultFileHandler` to be able to run some of the tests (the ones involving 'INCLUDE').

Also, see `InkParse.ts` below, no default file handler was registered. I'm not certain what was the original intention, maybe eventually assign a different handler by default, depending on the execution environment?

Reply:
When packaging for browser use (the main use case imo), it did not really make sense to assign the Posix/Filesystem handler, this is the main reason I named it that way (Default would not actually be default !).
Moreover, it creates a dependancy to node-fs that is in turn not resolvable for the web package.

Reply:
Makes sense! Do you think there's a way we could have a different default handler depending on the environment in InkParser?

I hope we can keep a default handler, otherwise, we need to flag this as an important difference and document it accordingly.

Reply:
Which tests were failing due to lack of default fileHandler ? Looks like you fixed it because when I comment it out it still works.

I think documentation is the way to go here, because even in a browser, it's not clear which "FileHandler" the user wants to use and I think it's a good idea to make people stop and think about how they want to handle multiple files (and if they don't it'll work without default anyway)

Reply:
> Which tests were failing due to lack of default fileHandler ? Looks like you fixed it because when I comment it out it still works.

[The tests explicitly use PosixFileHandler](https://github.com/y-lohse/inkjs/pull/939/files#diff-d3f98f7c5f5e3b3c64662aef9400b1310a81b137aaf1e7c3474bafe05467ec67R133) 😇 

> I think documentation is the way to go here, because even in a browser, it's not clear which "FileHandler" the user wants to use and I think it's a good idea to make people stop and think about how they want to handle multiple files (and if they don't it'll work without default anyway)

Alright, we need to replace that by a check and an explicit error then. Let me update the PR!